scripts/init.sql

```diff
@@ -1,289 +1,228 @@
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "name" varchar(255) NOT NULL,
+    "price" numeric(10, 2) NOT NULL,
+    "link" varchar(2000) NOT NULL,
+    "retailer" varchar(255) NOT NULL,
+    "retailer_product_id" varchar(255) NOT NULL,
+    "meta_data" json,
+    "image_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+);
+
+CREATE TABLE "messages" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "content" varchar(500) NOT NULL,
+    "sender_id" uuid NOT NULL,
+    "wishcard_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+);
+
+CREATE TABLE "orders" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "status" integer NOT NULL DEFAULT 0,
+    "delivery_date" timestamp,
+    "tracking_info" varchar(500),
+    "donor_id" uuid NOT NULL,
+    "child_id" uuid NOT NULL,
+    "item_id" uuid NOT NULL,
+    "agency_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
+    "updated_at" timestamp
+);
+
+CREATE TABLE "users" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "first_name" varchar(255) NOT NULL,
+    "last_name" varchar(255) NOT NULL,
+    "email" varchar(255) NOT NULL,
+    "password" varchar(100) NOT NULL,
+    "role" integer NOT NULL,
+    "login_mode" integer NOT NULL,
+    "bio" varchar(500),
+    "is_verified" boolean NOT NULL DEFAULT false,
+    "image_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
+    "updated_at" timestamp,
+    "deleted_at" timestamp
+);
+
+CREATE TABLE "verification_tokens" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "token" varchar(255) NOT NULL,
+    "type" integer NOT NULL,
+    "expiration" timestamp NOT NULL,
+    "user_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+);
+
+CREATE TABLE "verifications" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "email_verified" boolean NOT NULL,
+    "user_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
+    "updated_at" timestamp
+);
+
+CREATE TABLE "wishcards" (
+    "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
+    "address_line_1" varchar(255) NOT NULL,
+    "address_line_2" varchar(255),
+    "city" varchar(100) NOT NULL,
+    "state" varchar(100) NOT NULL,
+    "country" varchar(100) NOT NULL,
+    "zip_code" varchar(50) NOT NULL,
+    "status" integer NOT NULL DEFAULT 0,
+    "created_by" uuid NOT NULL,
+    "agency_id" uuid NOT NULL,
+    "child_id" uuid NOT NULL,
+    "item_id" uuid NOT NULL,
+    "image_id" uuid NOT NULL,
+    "order_id" uuid NOT NULL,
+    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
+    "updated_at" timestamp
+);
+
+ALTER TABLE "verifications" ADD CONSTRAINT "verifications_user_id_fkey" FOREIGN KEY ("user_id") REFERENCES "users" ("id") ON DELETE RESTRICT;
```

Comment:
@Enubia Have you confirmed that Kysely is able to pick up on these foreign key constraints since they are done outside of the initial table schema?

Reply:
The types are generated by introspection of the created database where the constraints are already on the table themselves 

This means that kysely is aware of them

Reply:
Sounds good to me, just wanted to make sure that was the case.

Thanks!